src/main/java/org/opendatakit/aggregate/client/externalserv/ServicesAdminServiceAsync.java

```diff
@@ -24,22 +24,15 @@
 
   void getExternalServices(String formid, AsyncCallback<ExternServSummary[]> callback);
 
-  void createFusionTable(String formId, ExternalServicePublicationOption esOption,
-                         String ownerEmail, AsyncCallback<String> callback);
+  void createFusionTable(String formId, ExternalServicePublicationOption esOption, String ownerEmail, AsyncCallback<String> callback);

```

Comment:
I dislike some of these newline changes because I often have two windows side by side or review on Github and now I have to scroll horizontally. I still prefer an 80-char width limit. But I won't push back too hard. Was this an automatic IntelliJ change?

Reply:
> way easier to read and change them when all members are displayed sequentially

That is very interesting! Even when writing and modifying code, I prefer shorter lines. But I can see your point. Anyway, not a big deal.